websites/M/Mahara-Tech/metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "605894319408283678",
+		"name": "eljooker"
+	},
+	"service": "Mahara-Tech",
+	"description": {
+		"en": "information technology learning platform"
+	},
+	"url": "maharatech.gov.eg",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/QRBVWsG.png",
```

Comment:
This logo is a bit blurry. Try to find or [create](http://waifu2x.udp.jp/) a higher-resolution version